src/components/status-badge/index.js

```diff
@@ -63,7 +63,13 @@ const getStatus = (status) => {
   return data;
 };
 
-export const StatusBadge = ({ children, type, customIconData, ...rest }) => {
+export const StatusBadge = ({
+  children,
+  type,
+  customIconData,
+  className = '',
```

Comment:
1. Add className to `StatusBadge.propTypes`
2. Add default value for className in `StatusBadge.defaultProps`  and not here